contrib/bloop/src/mill.contrib.bloop/BloopImpl.scala

```diff
@@ -92,13 +92,13 @@ class BloopImpl(ev: () => Evaluator, wd: Path) extends ExternalModule { outer =>
     * that does not get invalidated upon sourcefile change. Mainly called
     * from module#sources in bloopInstall
     */
-  def moduleSourceMap: Target[Map[String, Seq[Path]]] = T {
+  def moduleSourceMap = T.input {
```

Comment:
I was thinking that the cache would be invalidated upon the build changing, but it appear to not be the case for global tasks. Making this an input to ensure that `install` picks up new modules every time the user adds one.